### PR TITLE
[WIP] Update template with a newer version of helmet

### PR DESCRIPTION
**Pull request by dhmlau**
Suggested by @bajtos , updating the template with a newer version of `helmet`. 
> We should update our project template to install the latest helmet version too, see https://github.com/strongloop/loopback-workspace/blob/8726d7ce7662051fe3236c1174d132c8ab038261/templates/projects/empty-server/data.js#L26

I've also updated the min version for `debug`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "dependencies": {
     "async": "^1.5.2",
     "cookie-parser": "^1.3.2",
-    "debug": "^2.2.0",
+    "debug": "^2.6.9",
```
Comment (bajtos) [inline package.json]: This is effectively a no-op, because `^2.2.0` will install the latest 2.x version, e.g. `2.6.9.` Did you perhaps mean `^3.1.0`?

**Comment by dhmlau** [comment]
@slnode  test please

**Comment by bajtos** [comment]
I suspect helmet changed the default behaviour or perhaps the configuration format between major versions and that's why our tests are failing after upgrade. I can look into this next week if you like - just let me know, @dhmlau.

**Comment by dhmlau** [comment]
@bajtos , sorry, i've been juggling with a few things and haven't got the chance to get to it.  Will try to get back to it tmr.

**Pull request closed without merging**